scripts/linux/buildAPIProxy.sh

```diff
@@ -0,0 +1,202 @@
+#!/bin/bash
+#https://github.com/Azure/iotedge/blob/iiot/scripts/linux/cross-platform-rust-build.sh
+###############################################################################
+# This script builds a static binary of the api-proxy-module
+###############################################################################
+
+set -e
+
+###############################################################################
+# Define Environment Variables
+###############################################################################
+# Get directory of running script
+ARCH=$(uname -m)
+DIR=$(cd "$(dirname "$0")" && pwd)
+TARGET=
+STRIP=
+SCRIPT_NAME=$(basename "$0")
+PROJECT=
+DOCKERFILE=
+DOCKER_IMAGENAME=
+DEFAULT_DOCKER_NAMESPACE="microsoft"
+DOCKER_NAMESPACE=${DEFAULT_DOCKER_NAMESPACE}
+BUILD_BINARIESDIRECTORY=${BUILD_BINARIESDIRECTORY:-$BUILD_REPOSITORY_LOCALPATH}
+PUBLISH_DIR=${BUILD_BINARIESDIRECTORY}/publish
+API_PROXY_DIR=${BUILD_REPOSITORY_LOCALPATH}/edge-modules/api-proxy-module
+BUILD_CONFIGURATION="release"
+BUILD_CONFIG_OPTION=
+
+###############################################################################
+# Function to obtain the underlying architecture and check if supported
+###############################################################################
+check_arch()
```

Comment:
Just curious, do you know why not directly pass in the value, e.g. "arm32v7"?

Reply:
It is a very good point. I though about doing that too. However, I just want to keep buildImage.sh and buildAPIProxy script as close as possible.